.github/workflows/ci.yaml

```diff
@@ -26,72 +26,6 @@ jobs:
         with:
           globs: "**/*.md"
 
```

Comment:
removed crev since it was failing due to, https://github.com/crev-dev/cargo-crev/issues/598 and it seems like there isn't a desire to pursue it at this time.

Reply:
:+1: